automation/test.sh

```diff
@@ -312,7 +312,8 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ sig-network ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-network\\]"
   elif [[ $TARGET =~ sig-storage ]]; then
-    export KUBEVIRT_E2E_FOCUS="\\[sig-storage\\]"
+    export KUBEVIRT_E2E_FOCUS="\\[sig-storage\\]|rook-ceph"
```

Comment:
Is the `|rook-ceph` needed?

Reply:
I was doing this because I saw that one of the tests that runs in context of rook-ceph does not have a sig-storage label. So I thought we would just be safe by adding this one. Maybe we should align in another PR, or we can also do this here. @awels WDYT? 

Pointer to the rook-ceph context here: https://github.com/kubevirt/kubevirt/blob/a3b51ae43590948ec5294213bd13c65a73d57e4d/tests/migration_test.go#L1816

Reply:
Can we mae `rook-ceph` a label in the test descriptions? Something like `[rook-ceph]`? I would then use this expession to match: `\\[sig-storage\\]|\\[rook-ceph\\]`. Otherwise lgtm.